app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+    def create
+        @user = User.new(user_params)
+
+        if @user.save
+            redirect_to articles_url, notice: "User was successfully created."
```

Comment:
Decided to redirect them to the articles#index because we don't have a profile page yet

Reply:
This temporary redirect is fine you don't have many other pages to redirect to.

Should the user be logged in automatically on account creation for serious applications? What are some reasons we might want to redirect them to the login page, or a page that suggests a verification email has been sent? I know your current implementation doesn't support that, but the assumption that it would go to the profile page got me thinking about what we should really be planning for.